docs/index.rst

```diff
@@ -46,12 +52,12 @@ License: BSD-3
 News
```

Comment:
Agreed. We could start with the 0.1 release, or leave one line mentioning the GSoC work by Jazmin and Brett.